src/applications/gi/components/vet-tec/VetTecCalculator.jsx

```diff
@@ -209,7 +209,9 @@ export class VetTecCalculator extends React.Component {
         <div className="usa-width-one-half medium-6 columns vads-u-padding--0">
           <div className=" your-estimated-benefits">
             <h3>Your estimated benefits</h3>
-            <i>{this.props.calculator.vetTecProgramName}</i>
+            <div className="program-name">
```

Comment:
I think the font-style utility in the design system will do the same thing: https://design.va.gov/utilities/font-style

 `vads-u-font-style--italic`